frontend/src/views/coverage/enrollment/ResidentPHN.vue

```diff
@@ -0,0 +1,79 @@
+<template>
```

Comment:
Move component to components folder as mentioned.

Reply:
Moved.